templates/view/view.js

```diff
@@ -16,7 +16,7 @@
                 url: '/',
                 views: {
                     'main@': {
-                        templateUrl: getTemplateUrl('Home'),
+                        templateUrl: 'Home.html',
```

Comment:
This is to work with the new gulpfile flow utilizing `gulp-angular-templatecache`.